package.json

```diff
@@ -68,7 +68,8 @@
     "postcss": "8.4.20",
     "prettier": "2.8.0",
     "nanoid": "3.3.4",
-    "loader-utils": "2.0.4"
+    "loader-utils": "2.0.4",
+    "@svgr/webpack": "6.5.1"
```

Comment:
this resolution was needed to fix `yarn audit`.

@gretanausedaite Can you test a deploy of the website in azure pipelines? There might be more things we need to add here because of `npm audit`

Reply:
You can manually run build from branch in ADO (Run pipeline -> change branch from main).
We can setup automatic build on PRs (not sure if needed).